tests/test_app.py

```diff
@@ -1,4 +1,193 @@
-import todo_app
+import pytest
+from todo_app.models.view_model import ViewModel
+from todo_app.data.trello_items import Item
+from datetime import datetime, timedelta
+from dotenv import load_dotenv, find_dotenv
 
```

Comment:
Shouldn't this file be called test_view_model.py?

It's totally unrelated to app.py

Reply:
Resolved in module_4 https://github.com/ianm1968/ToDoTake2/commit/10760d33ef90f88f52dcb05aac6db3d48578f041